password-rotation/README.md

```diff
@@ -0,0 +1,43 @@
+# password-rotation-runner
+
+A Terraform module which deploys a scheduled ECS task on a cron schedule.  This task runs an application which pulls a test user spreadsheet from S3, rotates the Enterprise Portal passwords for the test users, updates the spreadsheet, and pushes it back to S3. To read more about the password rotation application, see the README (TBD)
+
+## Usage
+See [variables.tf](variables.tf) for variable descriptions
+```
+module "password-rotation" {
+  source      = "github.com/CMSgov/portal-test-user-manager//password-rotation
+  
+  app_name    = ""              // defaults to "password-rotation"
+  environment = "dev"
+  task_name   = ""               // defaults to "scheduled-runner"
+
+  repo_url                 = "156322662943.dkr.ecr.us-east-1.amazonaws.com/password-rotation"
+  image_tag                = "" // defaults to "latest"
+  ecs_vpc_id               = "" 
+  ecs_subnet_ids           = [] 
+  schedule_task_expression = "" // defaults to "0 8 1 * ? *", monthly on the first day of the month at 8am UTC (3am UTC-5)
```

Comment:
one way that can be nice for showing defaults is to do it like this:
```
  # image_tag = "latest"
  # schedule_task_expression = "0 8 1 * ? *" # monthly on the first day of the month at 8am UTC (3am UTC-5)
```
Then it is easy for the user of this example to uncomment and change lines where they don't like the default, and easy for them to leave it if they do want the default.

Also, use "#" consistently for terraform comments?